### ISSUE TITLE
Accessibility bug

### ISSUE THREAD

**Issue by Konserwa203** [FormidableLabs/nuka-carousel#859]
Hi, I'd like to report one accessibility bug:

1) There is a wrong property on paging-items: aria-selected

According to documentation there should be aria-pressed property instead.
https://www.levelaccess.com/how-not-to-misuse-aria-states-properties-and-roles/

**Comment by ValGeorgiev** [comment]
Hi @Konserwa203, thanks for raising this with us. I am not 100% convinced that `aria-pressed` is the more appropriate property for the paging items. As in the docs is described, `aria-selected` is `This attribute is used to indicate which elements within single-selection and multiple-selection composite widgets are selected.` [link](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected). While `aria-pressed` is for toggle buttons only. I know it's a hard call here, happy to hear more arguments.
